lib/ipmi_main.c

```diff
@@ -854,8 +854,7 @@ ipmi_main(int argc, char ** argv,
 
 	/* load the IANA PEN registry */
 	if (ipmi_oem_info_init()) {
-		lprintf(LOG_ERR, "Failed to initialize the OEM info dictionary");
-		goto out_free;
+		lprintf(LOG_WARN, "Failed to initialize the OEM info dictionary");
```

Comment:
Looks like the function may not fail now and can be made `void`.
The warning should be moved into that function instead.